### DISCUSSION TITLE
just api - Could not find the file /rails/public/assets/. in container demo_api-web-assets

### DISCUSSION THREAD

**Discussion by fabriciocarboni** [basecamp/kamal#1211] [Q&A] [answered]
Trying kamal for the first time and it seems not finding assets dir.
```
ERROR (SSHKit::Command::Failed): Exception while executing on host 198.199.68.149: mkdir exit status: 1
mkdir stdout: 10d593062d764926b0476e218e7098f995fee25c28ee948761f24d91ee9d9285
mkdir stderr: Error response from daemon: Could not find the file /rails/public/assets/. in container demo_api-web-assets
```
To overcome this I had to fake create an asset directory in `public/assets/fake.txt` 🤣
Is it possible to skip this in kamal as this app is just an api?

**Comment by jrroshan** [comment, marked as answer]
> Trying kamal for the first time and it seems not finding assets dir.
> 
> ```
> ERROR (SSHKit::Command::Failed): Exception while executing on host 198.199.68.149: mkdir exit status: 1
> mkdir stdout: 10d593062d764926b0476e218e7098f995fee25c28ee948761f24d91ee9d9285
> mkdir stderr: Error response from daemon: Could not find the file /rails/public/assets/. in container demo_api-web-assets
> ```
> 
> To overcome this I had to fake create an asset directory in `public/assets/fake.txt` 🤣 Is it possible to skip this in kamal as this app is just an api?

```# Bridge fingerprinted assets, like JS and CSS, between versions to avoid
# hitting 404 on in-flight requests. Combines all files from new and old
# version inside the asset_path.
asset_path: /rails/public/assets```

you can comment out this in deploy.yml

**Reply by fabriciocarboni** [reply]
Thank you!
